client/package.json

```diff
@@ -121,7 +121,7 @@
     "tslint-config-standard": "^9.0.0",
     "typescript": "~4.1.3",
     "video.js": "^7",
-    "videojs-contextmenu-ui": "^5.0.0",
+    "videojs-contextmenu-pt": "^5.4.1",
```

Comment:
Did you try to make a PR on the origin repository?

Reply:
No

Reply:
@Chocobozzz I opened a PR on brightcove's repository